gradle-baseline-java/src/main/groovy/com/palantir/baseline/tasks/CheckJUnitDependencies.java

```diff
@@ -47,7 +56,7 @@ public final void validateDependencies() {
                 .getPlugin(JavaPluginConvention.class)
                 .getSourceSets()
                 .forEach(ss -> {
-                    if (ss.getName().equals("main")) {
+                    if (ss.getName().equals(IGNORE_MAIN_CONFIGURATION)) {
```

Comment:
Not that it's every going to change, but there's https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/SourceSet.html#MAIN_SOURCE_SET_NAME

Reply:
done